### ISSUE TITLE
TypeError: this.getGridField is not a function

### ISSUE THREAD

**Issue by elliot-sawyer** [silvershop/silverstripe-hasonefield#31]
Issue raised on https://github.com/gorriecoe/silverstripe-linkfield/issues/11. It seems to be related to the recent removal of entwine and jquery. Reverting from 3.0.2 to 3.0.1 resolves the error.

**Comment by elliot-sawyer** [comment]
LinkField uses the HasOneField to manage links. 

Steps to Reproduce on SS4.3:
1. Add 2 new LinkFields to your getCMSFields()
2. Create a new Link in one of them
3. In the other, use the autocomplete to find and select the Link you just created
4. Click "Link existing".  Observe the console error.
5. Refresh the page. Observe the Gridfield still managed to link the record. 

Code: 

```php
    use SilverStripe\CMS\Model\SiteTree;
    use gorriecoe\LinkField\LinkField;
    use gorriecoe\Link\Models\Link;
    class Page extends SiteTree
    {
        private static $db = [];

        private static $has_one = [
            'Link1' => Link::class,
            'Link2' => Link::class,
        ];

        public function getCMSFields()
        {
            $fields = parent::getCMSFields();

            $fields->addFieldsToTab(
                'Root.Main',
                [
                    LinkField::create(
                        'Link1',
                        'Link 1',
                        $this
                    ),
                    LinkField::create(
                        'Link2',
                        'Link 2',
                        $this
                    ),
                ]
            );

            return $fields;
        }
    }
```

**Comment by christopherdarling** [comment]
Same issue here unfortunately. When selecting an existing record through the autocompleter, I see the console error mentioned above and the value doesn't update so looks as though the relation hasn't been updated.

I'm running SS 4.3.3 and the latest version of this module (3.0.3).

**Comment by sparkcom** [comment]
I got exactly the same problem and have to go back to 3.0.1 as well.

**Comment by mlewis-everley** [comment]
I am getting this issue as well, have reverted to 3.0.1, but I cannot see why this version works but 3.0.3 doesn't.

**Comment by wilr** [comment]
Fixed in #44
